osd/osd_too_old_upgrade_needed.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Warning",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: Upgrade cluster",
+    "description": "Your cluster is running an old OSD version. Please upgrade to a more recent minor version of OSD as soon as possible. Failing to upgrade will result in your cluster becoming unsupported and SRE monitoring of it being disabled, per our Life Cycle Policy, available here – https://access.redhat.com/support/policy/updates/openshift/dedicated",
```

Comment:
s/old/unsupported
s/more recent minor version/supported version

overall I would not phrase it like that because it doesn't give a timeline for the out of support part
either it's already out of support or it will become out of support in x days if nothing is happening

Reply:
Not sure it can be "unsupported", since this message should be sent during the 30 day period, while we still haven't pulled support from those clusters. "soon to be unsupported"? 

As to the rest, I agree, but to rephrase it like you'd want, I'd need a date, which I don't have, that's why I phrased it vaguely. Hmm, maybe on each minor release actually explicitly figure out the new cutoff date, update this message with it and then send it out? This PR needs to wait for us to actually figure out what cutoff date we want to use then.

Then summary would be "Action required: Upgrade cluster before 2021-99-99".

Reply:
Ok, applied these changes.